internal/blockchain/bifactory/factory.go

```diff
@@ -32,9 +32,9 @@ var pluginsByName = map[string]func() blockchain.Plugin{
 	(*fabric.Fabric)(nil).Name():     func() blockchain.Plugin { return &fabric.Fabric{} },
 }
 
-func InitPrefix(prefix config.Prefix) {
+func InitConfig(section config.Section) {
```

Comment:
nit: looks like you named this var `config` instead of `section` in all the other factories.